INSTALL.md

```diff
@@ -39,7 +39,7 @@
 2. Add the plugin to the list of plugins for Oh My Zsh to load (inside `~/.zshrc`):
 
     ```sh
-    plugins=(zsh-autosuggestions)
+    plugins=( [plugins...] zsh-autosuggestions)
```

Comment:
Can we write this in such a way that the `[plugins ...]` part can't be  confused for actual code? Could we do something like:
```
plugins+=(zsh-autosuggestions)
```

Or maybe this?

```zsh
plugins=(
  # Other plugins...
  zsh-autosuggestions
)
```

Reply:
I think second option is better, also more similar to other plugins.

I have updated pr accordingly